itou/settings_viewer/views.py

```diff
@@ -9,9 +9,9 @@
 def get_settings_list():
     """Build the list of (setting, value, is_default_value)."""
     # Code heavily inspired from django.core.management.commands.diffsettings
-    current_settings = settings._wrapped.__dict__
+    settings_keys = dir(settings)
     setting_cleanser = debug.SafeExceptionReporterFilter().cleanse_setting
-    return [(k, setting_cleanser(k, v)) for k, v in sorted(current_settings.items()) if not k.startswith("_")]
+    return [(k, setting_cleanser(k, getattr(settings, k))) for k in sorted(settings_keys) if not k.startswith("_")]
```

Comment:
```suggestion
    return [(k, setting_cleanser(k, getattr(settings, k))) for k in sorted(dir(settings)) if not k.startswith("_")]
```
Tant qu'on y est. Et ça ressemblera encore un peu plus au code de diffsettings :sweat_smile: